src/data/set/finite.lean

```diff
@@ -396,6 +396,24 @@ let ⟨I, Ifin, hI⟩ := finite_subset_Union tfin h in
         exact H }
     end⟩
 
+instance nat.fintypeIio (n : ℕ) : fintype (Iio n) :=
+fintype.of_finset (finset.range n) $ by simp
+
+lemma seq_of_forall_finite_exists  {γ : Type*}
```

Comment:
```suggestion
/--
If `P` is some relation between terms of `γ` and sets in `γ`, 
such that every finite set `t : set γ` has some `c : γ` related to it, 
then there is a recursively defined sequence `u` in `γ` 
so `u n` is related to the image of `{0, 1, ..., n-1}` under `u`.

(We use this later to show sequentially compact sets
are totally bounded.)
-/
lemma seq_of_forall_finite_exists  {γ : Type*}
```

(It just took me too long to decode what the statement was,
so I thought I'd give future me some help.)